src/Settings.php

```diff
@@ -43,6 +43,48 @@ public function renderFormFields() {
         'description' => __('If checked, the Trusted Shops responsive banner will be hidden on mobile devices.', Plugin::L10N),
         'desc_tip' => TRUE,
       ],
+      'trusted_shops/display_product_stars' => [
+        'title' => __('Display product stars', Plugin::L10N),
+        'type' => 'checkbox',
+        'description' => __('If checked, Trusted Shops stars will be displayed in product pages.', Plugin::L10N),
+        'desc_tip' => TRUE,
+      ],
+      'trusted_shops/display_product_reviews' => [
+        'title' => __('Display product reviews', Plugin::L10N),
+        'type' => 'checkbox',
+        'description' => __('If checked, Trusted Shops reviews will be displayed in product pages.', Plugin::L10N),
+        'desc_tip' => TRUE,
+      ],
+      'trusted_shops/product_review_box_bordercolor' => [
+        'title' => __('Product reviews box border color', Plugin::L10N),
+        'type' => 'color',
+        'description' => __('Color of product review box borders.', Plugin::L10N),
```

Comment:
IMHO this and the following descriptions are not needed. The title is already pretty self-explanatory.